dotCMS/src/main/java/com/dotmarketing/fixtask/tasks/FixTask00095DeleteOrphanRelationships.java

```diff
@@ -0,0 +1,173 @@
+package com.dotmarketing.fixtask.tasks;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.sql.SQLException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import com.dotcms.repackage.com.thoughtworks.xstream.XStream;
+import com.dotcms.repackage.com.thoughtworks.xstream.io.xml.DomDriver;
+import com.dotmarketing.beans.FixAudit;
+import com.dotmarketing.beans.Inode;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.HibernateUtil;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.fixtask.FixTask;
+import com.dotmarketing.portlets.cmsmaintenance.ajax.FixAssetsProcessStatus;
+import com.dotmarketing.portlets.cmsmaintenance.factories.CMSMaintenanceFactory;
+import com.dotmarketing.util.ConfigUtils;
+import com.dotmarketing.util.Logger;
+
+
```

Comment:
Some Javadoc explaining the reasons why this task is helpful would be great.

Reply:
I'm on it. thanks for the heads up!